webapp/shop/cube/views.py

```diff
@@ -355,3 +361,103 @@ def cred_syllabus_data(**kawrgs):
         syllabus_data=syllabus_data,
         exam_name=exam_name,
     )
+
+
+@shop_decorator(area="cube", permission="user_or_guest", response="html")
```

Comment:
Let's put this page behind SSO since users should already have an Ubuntu One account when landing here. (It will also allow us to link survey responses to users.)

Reply:
But do we want that? Shouldn't these be anonymous?

Reply:
Anonymity is helpful for making people feel more free in their responses, but I'd like to be able to correlate test-taker to surveyor. Some people may give us a bad review because of their performance, and it would be helpful to assess for that.